src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -72,6 +74,19 @@
 
           <% } %>
 
+    <p>Status: </p>
+    <p><font size="4">${status}</font></p>
+
+       <% if (statusUrl != ""){ %>
+
+         <% if ( currentUser.getName().equals(displayUser.getName()) ){ %>
```

Comment:
Make this if statement wrap the entire logic. Don't forget the close `<%> } <%>`. Same for Bio